### ISSUE TITLE
Traffic Manager needs way to configure IntervalInSeconds, ToleratedNumberOfFailures and TimeoutInSeconds

### ISSUE THREAD

**Issue by TeaTable** [Azure/azure-libraries-for-net#518]
Currently in the library I don't find a way to configure Traffic Manager monitor with intervalInSeconds, toleratedNumberOfFailures or timeoutInSeconds:

"monitorConfig": {
      "profileMonitorStatus": "Inactive",
      "protocol": "HTTP",
      "port": 80,
      "path": "/testpath.aspx",
      "intervalInSeconds": 30,
      "toleratedNumberOfFailures": 3,
      "timeoutInSeconds": 10
    }

The available interfaces I can find are:

Microsoft.Azure.Management.TrafficManager.Fluent.TrafficManagerProfile.Definition.IWithCreate WithHttpMonitoring();

Microsoft.Azure.Management.TrafficManager.Fluent.TrafficManagerProfile.Definition.IWithCreate WithHttpMonitoring(int port, string path);

Microsoft.Azure.Management.TrafficManager.Fluent.TrafficManagerProfile.Definition.IWithCreate WithHttpsMonitoring();

Microsoft.Azure.Management.TrafficManager.Fluent.TrafficManagerProfile.Definition.IWithCreate WithHttpsMonitoring(int port, string path);

**Comment by jsukhabut** [comment]
This is becoming important for us now to use the Fast Failover feature.

https://azure.microsoft.com/en-us/updates/fast-failover-and-tcp-probing-in-azure-traffic-manager/

**Comment by jsukhabut** [comment]
@hovsepm  I am currently working on adding this to the library.

I am also going to add enabled/disable Traffic View feature.

**Comment by yaohaizh** [comment]
@weidongxu-microsoft  please take a look at this issue after the traffic manager is updated.

**Comment by weidongxu-microsoft** [comment]
Fixed in 1.29. To be released on Dec. 9th.

Sample code:

```
                    var trafficManagerProfile = azure.TrafficManagerProfiles.Define(profileName)
                            ...
                            .WithHttpsMonitoring()
                            .WithFastFailover(10, 5)
```
